demos/TicTacToe/setup_demo.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+# coding=utf-8
+
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+import fileinput
+import glob
+import logging
+import os
+import subprocess
+import sys
+
+
+# The setup_demo.py script directory.
+ROOT_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+def main():
+  """The main function."""
+  # The root logger of the hierarchy.
+  logger = logging.getLogger()
+  logger.setLevel(logging.DEBUG)
+
+  # Add a StreamHandler to log to stdout.
+  stream_handler = logging.StreamHandler(sys.stdout)
+  stream_handler.setLevel(logging.DEBUG)
+  formatter = logging.Formatter(
+      "%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+  stream_handler.setFormatter(formatter)
+  logger.addHandler(stream_handler)
```

Comment:
Super minor note but I would move all the logging stuff into another function and have a one liner for all logging related stuff in `main`. As a user who is trying to figure out this script, a quick glance at main should tell me at a high level what is happening.

Reply:
Added a logger_setup() function do implement this change.